data/src/main/java/com/playone/mobile/data/source/BufferooDataStoreFactory.kt

```diff
@@ -16,8 +16,12 @@ open class BufferooDataStoreFactory @Inject constructor(
      * Returns a DataStore based on whether or not there is content in the cache and the cache
      * has not expired
      */
-    open fun retrieveDataStore() =
-        if (bufferooCache.isCached() && !bufferooCache.isExpired())  retrieveCacheDataStore() else retrieveRemoteDataStore()
+    open fun retrieveDataStore() = if (bufferooCache.isCached() && !bufferooCache.isExpired()) {
+        retrieveCacheDataStore()
```

Comment:
Since the scope is in `if` not in the function so I think it is better to do one more indent for it.
for instance:
```
open fun retrieveDataStore() = if (bufferooCache.isCached() && !bufferooCache.isExpired()) {
               retrieveCacheDataStore()
        } else {
              ....
        }
```